changelog/2024-05-31T20_14_29+02_00_fix_bitvector_shifts

```diff
@@ -0,0 +1 @@
+Fixes (+>>.) and (.<<+) such that they are compliant with (+>>) and (<<+) for vectors of zero length.
```

Comment:
Could you please describe what they do for vectors of zero length? It's fine to note that they do the same as their vector counterparts, but the natural question that would boil up is: so what do they do? :-)